src/models/GOKU.jl

```diff
@@ -5,8 +5,21 @@
 
 struct GOKU <: LatentDE end
 
+@doc raw"""
+    apply_feature_extractor(encoder, x)
+Converts a batch of the initial high-dimensional data into lower-dimensional data (i.e. extracts features)
+# Arguments
+encoder: Has type Encoder{GOKU} or Encoder{LatentODE}\
```

Comment:
I would avoid saying mentioning GOKU and LatentODE here. I would say:

encoder: Structure of type Encoder containing all the encoders layers.

Reply:
This was my solution to not having separate documentation for ::Encoder{GOKU} and ::Encoder{LatentODE} (see below comment).  If keep my solution, how about "encoder: Structure of type Encoder containing all the encoders layers, has type Encoder{GOKU} or Encoder{LatentODE}"?  If separate documentation for each encoder type, I will just do as you suggested here.

Reply:
I would keep only the most general  case for now, just "encoder: Structure of type Encoder containing all the encoders layers". Later I'll see how it is displayed and what is the right way of mentioning all the different signatures.